### DISCUSSION TITLE
Moonraker and Klipper not talking

### DISCUSSION THREAD

**Discussion by nutu-blip** [Arksine/moonraker#661] [General]
Not sure if I'm in the right place but I just installed MainSail, MoonRaker and Klipper on to a rpi4, Klipper firmware also flashed to my printer. MainSail indicates that MoonRaker and Klipper are not communicating and when I use Putty to check for the mcu id it shows "  usb-1a86_USB_Serial-if00-port0 ". I'm new to all of this and would really appreciate any help i can get. I've attached the MoonRaker log if it helps.

[moonraker (10).log](https://github.com/Arksine/moonraker/files/11583157/moonraker.10.log)

**Comment by chinifabio** [comment]
I guess you didn't flash correctly the firmware on your printer, otherwise the mcu id  would be something like that 'usb-Klipper_sam4e8e_00313753384C39323134303431303231-if00'.
